README.md

```diff
@@ -43,20 +43,24 @@ crash*. It doesn't necessarily mean it will run well on your system ;) It featur
 ### Using the CLI
 To make things easier, a CLI has been developed. First, clone the repository in a directory of your
 choosing. You can run it by entering `python main.py` and the CLI will guide you through the
-process. Please take a look at [important notes](#important-notes) before you continue.
+process. Please take a look at [important notes](#important-notes) before you continue. 
+**This is the recommended method at the moment.**
 
 ### Manually
+Manual installation currently isn't very sophisticated, but it can be done.
 1. To get started, clone the repository in a directory of your choosing. **Note: this will be where
    your installation and media will be as well, so think about this a bit.**
-2. Copy `.env.sample` to a real `.env` by running `$ cp .env.sample .env`.
-3. Set the environment variables to your liking. Note that `ROOT_DIR` should be the directory you
-   have cloned this in.
+2. ~~Copy `.env.sample` to a real `.env` by running `$ cp .env.sample .env`.~~
+3. ~~Set the environment variables to your liking. Note that `ROOT_DIR` should be the directory you
+   have cloned this in.~~
```

Comment:
I figured that this was a remainder of a previous implementation because docker-compose didn't use those variables anywhere, I should've asked you about that first